static/app/views/projectInstall/createProject.tsx

```diff
@@ -141,191 +227,49 @@ class CreateProject extends Component<Props, State> {
-      });
-
-      // Only log this if the error is something other than:
-      // * The user not having access to create a project, or,
-      // * A project with that slug already exists
-      if (err.status !== 403 && err.status !== 409) {
-        Sentry.withScope(scope => {
-          scope.setExtra('err', err);
-          scope.setExtra('props', this.props);
-          scope.setExtra('state', this.state);
-          Sentry.captureMessage('Project creation failed');
-        });
-      }
-    }
-  };
-
-  setPlatform = (platformKey: PlatformName | null) => {
-    if (!platformKey) {
-      this.setState({platform: null, projectName: ''});
-      return;
-    }
+          organization={organization}
+          showOther
+        />
+        <IssueAlertOptions onChange={updatedData => setAlertRuleConfig(updatedData)} />
 
-    this.setState(({projectName, platform}) => {
-      // Avoid replacing project name when the user already modified it
-      const userModifiedName = projectName && projectName !== platform;
-      const newName = userModifiedName ? projectName : platformKey;
-
-      return {
-        platform: platformKey,
-        projectName: slugify(newName),
-      };
-    });
-  };
-
-  render() {
-    const {platform, error} = this.state;
-
-    return (
-      <Fragment>
-        {error && <Alert type="error">{error}</Alert>}
-
-        <div data-test-id="onboarding-info">
-          <Layout.Title withMargins>{t('Create a new project in 3 steps')}</Layout.Title>
-          <HelpText>
-            {tct(
-              'Set up a separate project for each part of your application (for example, your API server and frontend client), to quickly pinpoint which part of your application errors are coming from. [link: Read the docs].',
-              {
-                link: (
-                  <ExternalLink href="https://docs.sentry.io/product/sentry-basics/integrate-frontend/create-new-project/" />
-                ),
-              }
-            )}
-          </HelpText>
-          <Layout.Title withMargins>{t('1. Choose your platform')}</Layout.Title>
-          <PlatformPicker
-            platform={platform}
-            defaultCategory={this.defaultCategory}
-            setPlatform={selectedPlatform =>
-              this.setPlatform(selectedPlatform?.id ?? null)
-            }
-            organization={this.props.organization}
-            showOther
-          />
-          <IssueAlertOptions
-            onChange={updatedData => {
-              this.setState({dataFragment: updatedData});
-            }}
-          />
-          {this.renderProjectForm()}
-        </div>
-      </Fragment>
-    );
-  }
+        {createProjectForm}
+      </div>
+    </Fragment>
+  );
 }
 
-// TODO(davidenwang): change to functional component and replace withTeams with useTeams
-export default withRouteAnalytics(
-  withApi(withSentryRouter(withOrganization(withTeams(CreateProject))))
-);
 export {CreateProject};
```

Comment:
should we just have this as a default export?

Reply:
I am trying to remove default exports